front/src/applications/operationalStudies/views/Scenario/components/Timetable/utils.ts

```diff
@@ -223,3 +225,46 @@ export const isValidPathfinding = (summaryTrain: SimulationSummary | undefined)
   }
   return true;
 };
+
+export const computeTimetablePackageName = (
+  trainScheduleSetName: string,
+  catalogueName?: string
+): string => (catalogueName ? `${catalogueName} | ${trainScheduleSetName}` : trainScheduleSetName);
+
+export const sortTrainScheduleSets = (
+  set1: TrainScheduleSet,
+  set2: TrainScheduleSet,
+  catalogueEntry: CatalogueEntry[]
+): number => {
+  const set1CatalogueId = set1.catalogue_entry_id;
+  const set2CatalogueId = set2.catalogue_entry_id;
+
+  const catalogueName1 = set1CatalogueId
```

Comment:
if `set1CatalogueId` is `0`, then the name is undefined and so the final sort is bad.
In the mock I use the `0` as an ID, but by checking PG default sequence, it should not occurs in production.
Don't know if we should fix it here ...  but I will change the mock system.